src/RegexColumnizer.UnitTests/RegexColumnizerTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using LogExpert;
+using Moq;
+using NUnit.Framework;
+
+namespace RegexColumnizer.UnitTests
+{
+    [TestFixture]
+    public class RegexColumnizerTests
+    {
+        [Test]
+        public void SplitLineLogRegexDoesNotMatchLine()
+        {
+            var columnizerConfig = new RegexColumnizerConfig
+            {
+                //We just need a regex that matches the line.
+                Expression = @"^(?'time'[\d]+)\s+(?'Message'.+)$",
+                Name = "Test regex"
+            };
+
+            var columnizer = new Regex1Columnizer();
+            columnizer.Init(columnizerConfig);
+            
+            var notMatchingLogLine = new TestLogLine(5, "Error in com.example.core");
+            
+            var parsedLogLine = columnizer.SplitLine(Mock.Of<ILogLineColumnizerCallback>(), notMatchingLogLine);
+            
+            // verify that the expected number of columns are created
```

Comment:
Every Assert should be its own testcase, if those are just different data-sets, add the data to the function header (see the examples in "LogExpert.Tests" project :)

Reply:
thanks for contributing 👍